manubot/cite/handlers.py

```diff
@@ -25,6 +25,25 @@
     "manubot.cite.wikidata.Handler_Wikidata",
 ]
 
+_infer_prefix_patterns = {
+    ("doi", "accession_pattern"),
+    ("doi", "shortdoi_pattern"),
+    ("pmc", "accession_pattern"),
+    ("pubmed", "accession_pattern"),
```

Comment:
Note what now happens with `manubot cite --md 1`

> 1. **Formate assay in body fluids: application in methanol poisoning.**
AB Makar, KE McMartin, M Palese, TR Tephly *Biochemical medicine* (1975-06) <https://www.ncbi.nlm.nih.gov/pubmed/1>
DOI: [10.1016/0006-2944(75)90147-7](https://doi.org/10.1016/0006-2944(75)90147-7) · PMID: [1](https://www.ncbi.nlm.nih.gov/pubmed/1)

The other patterns are unlikely to match user-defined keys. But since PubMed IDs can start at 1, this pattern will pick up user defined citekeys that employ a simple numbering scheme. This wouldn't be catastrophic, since these citekeys would have to have metadata provided in a bibliography. However both of the citekeys would get `pubmed:` prefixed while using the metadata from bibliography.

Another options would be to use a stricter regex for inferring a pubmed prefix. For example, six digit minimum.

References:

- https://phabricator.wikimedia.org/T154905

Reply:
Because there are valid PMIDs that are shorter than the six digit minimum, then I think Manubot should support those the same as longer PMIDs.  Using numbering scheme for user defined citekeys isn't a great idea even if it is valid.